src/components/toast/toast.component.js

```diff
@@ -19,20 +19,26 @@ class Toast extends React.Component {
     );
   }
 
-  /** Content rendered for dismiss X */
-  get dismissIcon() {
-    if (this.props.onDismiss) {
-      return (
-        <DismissButton
-          data-element='close'
-          onDismiss={ this.props.onDismiss }
-          type='close'
-          variant={ this.props.variant || this.props.as }
-        />
-      );
-    }
-    return null;
-  }
+  closeIcon = () => {
```

Comment:
I think it can be just class method without an arrow. Just to be consistent in this class

Reply:
fixed: 979d4be